Custom/events/JobStats/JobStats.py

```diff
@@ -1,10 +1,11 @@
 ###############################################################
 # Imports
 ###############################################################
-from System import TimeSpan
+#from System import TimeSpan

```

Comment:
comment out or just needs deleting as now un-used?

Reply:
removed the commented lines.